src/core/pandoc/format-extension.ts

```diff
@@ -13,14 +14,10 @@ export const kPandocExtensions = [
   "-abbreviations",
   "+all_symbols_escapable",
   "-all_symbols_escapable",
-  "+amuse",
```

Comment:
Why did we lose a bunch of the format extensions? (is this auto-generated?). It's important that we get these right as I believe they are cleaved out of the namespace for `+` variants. @dragonstyle anything to add here?

Reply:
These are auto-generated by `package/src/common/update-pandoc.ts` now, so pandoc 3.1 must not be reporting them.

Reply:
I wonder if that is a bug in pandoc's reporting then? (e.g. it seems hard to believe that they got rid of `sourcepos`). I think we should dig into this a bit and perhaps even be conservative (and just add items) if we can't figure out what's going on (this assumes that the main use of this is masking out potential `+` variations, @dragonstyle will know for sure).

Reply:
We are using these for parsing the format strings, so the biggest downside of including them will be that they could technically be allowed as user modifiers but will be treated as Pandoc variants. And it does appear that for whatever reason, Pandoc is no longer reporting these as valid options (though `sourcepos` is supported as a reader option).

Is the issue perhaps that these are reader options vs writer options? 

I'm happy to open an issue with Pandoc if that is the right next step as well. In the meantime we definitely can just hard append these options in the generator - I'll make a commit that does that for now.

Reply:
Something definitely has been lost here compared to v2.19.2 (as I'm sure they haven't actually gotten rid of all of these options). It very well could be reader vs. writer and before they were giving us both but now only writer? In any case worth digging in to before we merge.

Reply:
(Commit to restore those variants here https://github.com/quarto-dev/quarto-cli/pull/4416/commits/5af06fb43fc5df87d61e2e669c654ea91417f0ee)